administration/src/util/translations/de.json

```diff
@@ -573,7 +573,8 @@
     "csvCreationError": "Etwas ist schiefgegangen beim Erstellen der CSV.",
     "freinetAgencyNotFound": "Diese Agentur konnte in Freinet nicht gefunden werden.",
     "freinetMultiplePersonError": "Der antragstellenden Person konnte keine Person in Freinet-Online eindeutig zugeordnet werden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell.",
-    "freinetApiNotReachableError": "Die eingegebenen Daten stimmen nicht eindeutig mit einer Person in Freinet-Online überein. Bitte prüfen Sie die Angaben oder nehmen Sie die Aktualisierung direkt in Freinet-Online vor."
+    "freinetApiNotReachableError": "Die eingegebenen Daten stimmen nicht eindeutig mit einer Person in Freinet-Online überein. Bitte prüfen Sie die Angaben oder nehmen Sie die Aktualisierung direkt in Freinet-Online vor.",
```

Comment:
🔧  This error message makes no sense. The key says api not reachable error and the text says that the data can not be related to a particular person.
We need here a more general message like
"Die Verbindung zu Freinet-Online konnte nicht hergestellt werden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell."

Reply:
okay i didn't read that. But it actually makes no sense whats standing there for the first part. (since this means that multiple persons were found. But in our case we want to show an error message if we can not establish a connection to the api)
The last part could be last technical like

`Die Verbindung zu Freinet-Online konnte nicht hergestellt werden. nehmen Sie die Aktualisierung direkt in Freinet-Online vor.`